metrics/cli/docs/features/supported_storage_version/01_supported_storage_version.md

```diff
@@ -0,0 +1,80 @@
+# Supported Storage Version
+> Feature description / User story.
+
+Improve Metrics Web application deployment process to include the supported storage version passing and storage version saving. 
+
+## Contents
+
+- [**Analysis**](#analysis)
+    - [Feasibility study](#feasibility-study)
+    - [Requirements](#requirements)
+    - [Landscape](#landscape)
+    - [Prototyping](#prototyping)
+    - [System modeling](#system-modeling)
+
+# Analysis
+> Describe general analysis approach.
+
+During the analysis section, we are going to define the requirements for this feature and choose whether to use the existing approach or develop a custom one to implement this feature.
+
+### Feasibility study
+> A preliminary study of the feasibility of implementing this feature.
+
+Since the `Metrics Web` application would receive updates after the initial release, we should have a convenient way of updating the `Metrics Web` application even if the database has any significant changes. To be able to update the application safely, we are going to introduce a storage version and a supported storage version for all `Metrics` applications. It will allow us to block the application from interaction with the persistent storage in case the application does not support the current storage version or the storage is currently updating. To simplify the `Metrics Web` application deployment process, we should update the `Metrics CLI` to make it set the supported storage version to the `Metrics Web` application.
```

Comment:
:readable:
Would it be better to make the section more granular?
```suggestion
Since the `Metrics Web` application would receive updates after the initial release, we should have a convenient way of updating the `Metrics Web` application, even if the database has no significant changes.

To be able to update the application safely, we are going to introduce a storage version and a supported storage version for all `Metrics` applications. This will allow us to block the application from interaction with the persistent storage in case the application does not support the current storage version or the storage is currently updating. 

To simplify the `Metrics Web` application deployment process, we should update the `Metrics CLI` to make it set the supported storage version to the `Metrics Web` application.
```